mantidimaging/gui/ui/spectrum_viewer.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>921</width>
-    <height>739</height>
+    <width>913</width>
+    <height>710</height>
```

Comment:
If still required after decreasing the width of the spinbox's, can we increase the default size of the window a little more to cover lower resolution screens? I get scroll bars with the initial window size with resolutions 1792 x 1120 x 1080, 1200 and 1080 x 1920
![image](https://github.com/mantidproject/mantidimaging/assets/32413847/7aa1ab35-4094-493c-a8f5-43f59723060d)